packages/docs/components/Card/Card.js

```diff
@@ -0,0 +1,41 @@
+import { Card, Text } from '@platzily-ui/components';
+import { createTheme, ThemeProvider, createStyleSheet } from '@platzily-ui/styling';
+
+const useStyleSheet = createStyleSheet(() => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  textCenter: {
+    textAlign: 'center',
+  },
+}));
+
+const header = (textCenter) => {
```

Comment:
Components should be named in upper camel case, lets go that approach instead, and pass textCenter not as a paramenter but as a prop.